src/unlink.js

```diff
@@ -89,10 +89,14 @@ module.exports = function unlink(config, args) {
   unlinkDependencyIOS(project.ios, dependency, packageName);
 
   const allDependencies = getDependencyConfig(config, getProjectDependencies());
+  const otherDependencies = filter(allDependencies, ({name}) => name !== packageName);
```

Comment:
Nice. The only thing is that we're officially support node@4.1+, so we can't use ES6 destructuring 😢